scripts/is-release-needed.sh

```diff
@@ -4,10 +4,13 @@
 PACKAGE_VERSION="$(node -p -e "require('./package.json').version")"
 PACKAGE_NAME="$(node -p -e "require('./package.json').name")"
 
-FOUND_VERSION=$(npm view $PACKAGE_NAME versions | grep $PACKAGE_VERSION)
+FOUND_VERSION=$(npm view $PACKAGE_NAME versions --json | grep -o "\"$PACKAGE_VERSION\"")
 
 if [ -z "$FOUND_VERSION" ]; then
     IS_NEW_VERSION=true
+else
+    IS_NEW_VERSION=false
 fi
 
-echo "is-release-needed=$(echo $IS_NEW_VERSION)"
+echo "is-release-needed=$IS_NEW_VERSION" >> $GITHUB_ENV
```

Comment:
Oh. Is this an alternative to GITHUB_OUTPUT? I guess, if it works, but I kinda like GITHUB_OUTPUT better because it seems less magical, more obvious what's happening.